docs/locators.md

```diff
@@ -0,0 +1,289 @@
+# Locators
+
+"Locators" in `wdi5` are referred to [as "Matchers" in OPA5](https://ui5.sap.com/#/api/sap.ui.test.matchers): they identify a UI5 control to retrieve and use for further (test-)work.  
+At the same time, locators are a subset [of a "Selector" in `wdi5`](usage#control-selectors).
+
+```javascript
+// this is the overall selector
+const identifier = {
+  wdio_ui5_key: "someCustomIdInWdi5Test",
+  forceSelect: true,
+  // this is the locator/matcher (sorry for not naming it as such 😬)
+  selector: {
+    i18NText: {
+      propertyName: "text",
+      key: "startPage.navButton.text"
+    },
+    controlType: "sap.m.Button",
+    viewName: "test.Sample.view.Main"
+  }
+}
+
+// get it
+const control = await browser.asControl(identifier)
+// use it
+await control.press()
+```
+
+Here are couple of samples to get you comfortable with `wdi5`'s/OPA5's locators/matchers.
+
+?> Note: all of them are essentially taken from the `/examples/ui5-js-app/webapp/test/e2e/**/*.test.js` files.
+
+## ancestor
+
+Get a "child" control from a "parent" (ancestor).
```

Comment:
applies also for the descendent matcher: as far as I know the optional parameter `direct` as described in the [API](https://sapui5.hana.ondemand.com/#/api/sap.ui.test.matchers.Ancestor) is currently not possible. So the locator searches all parents/descendents in the hierarchy and not only the next one in the DOM Tree

Reply:
added the hint on both ancestor and descendant